Application/mobile/app_code/lib/pages/container/details_state.dart

```diff
@@ -116,7 +116,7 @@ class ContainerDetailsState extends State<ContainerDetailsPage> {
               mainAxisAlignment: MainAxisAlignment.center,
               children: [
                 Text(
-                  '$_localization par $_owner',
+                  '$_city par $_adress',
```

Comment:
Can't understand that text, would it looks like "Nantes par place Graslin" for example ?